packages/block-library/src/latest-posts/index.php

```diff
@@ -48,14 +48,6 @@ function render_block_core_latest_posts( $attributes ) {
 	$block_core_latest_posts_excerpt_length = $attributes['excerptLength'];
 	add_filter( 'excerpt_length', 'block_core_latest_posts_get_excerpt_length', 20 );
 
-	$filter_latest_posts_excerpt_more = static function ( $more ) use ( $attributes ) {
-		$use_excerpt = 'excerpt' === $attributes['displayPostContentRadio'];
```

Comment:
As mentioned in https://github.com/WordPress/gutenberg/issues/55026#issuecomment-1746246347

I think I used this so that choices of the user in the editor were reflected on the frontend, even if those changes overrode theme custom text.

Theme custom excerpt text would still display if displayPostContentRadio was not set to 'excerpt' in the Latest Posts block attributes. However, I believe we were missing the editor's "excerptLength attribute is less than the currently-set excerpt length" [condition](https://github.com/WordPress/gutenberg/blob/4b804ac39b34332115562817c717ff4b7206c36e/packages/block-library/src/latest-posts/edit.js#L476-L476) in index.php.

What should prevail here?